osgp/protocol-adapter-iec60870/osgp-protocol-simulator-iec60870/src/main/java/org/opensmartgridplatform/simulator/protocol/iec60870/server/Iec60870ASduHandler.java

```diff
@@ -0,0 +1,38 @@
+package org.opensmartgridplatform.simulator.protocol.iec60870.server;
+
+import java.io.IOException;
+
+import javax.annotation.PostConstruct;
+
+import org.openmuc.j60870.ASdu;
+import org.openmuc.j60870.Connection;
+import org.openmuc.j60870.TypeId;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public abstract class Iec60870ASduHandler {
+
+    private final Logger logger;
+
+    @Autowired
+    protected Iec60870ASduHandlerMap iec60870ASduHandlerMap;
+
+    private TypeId typeId;
+
+    public Iec60870ASduHandler(final Logger Logger, final TypeId typeId) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS00117)